README.md

```diff
@@ -510,7 +510,12 @@ You can specify config defaults that will be applied to every request.
 
 ```js
 axios.defaults.baseURL = 'https://api.example.com';
+axios.defaults.headers.common['User-Agent'] = /* eg. 'AppName/1.0.0' */;
```

Comment:
I suggest to remove this example. See https://github.com/axios/axios/issues/1231#issuecomment-575158262.

Reply:
Thanks, that's a good point. As the User-Agent example isn't the reason for this PR I agree it should be removed since it's a little more complicated than the one-liner.